src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write code here
+
+const listaUl = document.querySelector('ul');
+const listaLi = listaUl.querySelectorAll('li');
+
+const posortowanaLista = [...listaLi].sort((a, b) => {
+  const salaryA = Number(a.getAttribute('data-salary').replace(/[$,]/g, ''));
+  const salaryB = Number(b.getAttribute('data-salary').replace(/[$,]/g, ''));
+
+  return salaryB - salaryA;
```

Comment:
The sort comparator function directly subtracts 'salaryB' from 'salaryA'. This works but consider adding a check to ensure that the values being subtracted are indeed numbers after the conversion from string to prevent unexpected behavior.